src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml

```diff
@@ -6,13 +6,6 @@ admin_features_index:
     _legacy_controller: AdminFeatures
     _legacy_link: AdminFeatures
 
-admin_features_horizontal_index:
```

Comment:
Hmm it's weird, this is supposed to be cleaned already in this PR https://github.com/PrestaShop/PrestaShop/pull/35191 🤔 Maybe your branch needs to be rebased?

Reply:
Yeah, but I think that as it's a resolvable change and the same code, git is not complaining. :-) It's an older PR.